pages/code.md

```diff
@@ -2,19 +2,51 @@
 title:  "The JuMP ecosystem"
 ---
 
-The JuMP project consists of the main [JuMP](https://github.com/juliaopt/JuMP.jl)
+The JuMP project consists of the main [JuMP](https://github.com/jump-dev/JuMP.jl)
 package, and a large collection of supporting Julia packages.
 
 The JuMP source code can be found at [https://github.com/jump-dev/JuMP.jl](https://github.com/jump-dev/JuMP.jl).
 
-Beneath JuMP, there is an abstraction layer called MathOptInterface ([source code](https://github.com/jump-dev/MathOptInterface.jl)). MathOptInterface defines an API that solvers
-should implement so that they can be used by JuMP, handles the automatic
-reformulation of problems via _bridges_, and has a large infrastructure for
-automatically testing solvers.
+## MathOptInterface
 
-At the bottom of the stack, there are solver wrappers. These packages wrap the
-solvers, which are often written in C or C++, and implement the API defined by
-MathOptInterface.
+Beneath JuMP, there is an abstraction layer called MathOptInterface 
+([source code](https://github.com/jump-dev/MathOptInterface.jl)). 
+
+MathOptInterface defines an API that solvers should implement so that they can 
+be used by JuMP, handles the automatic reformulation of problems via _bridges_, 
+and has a large infrastructure for automatically testing solvers.
+
+## Solvers and solver-wrappers
+
+At the bottom of the stack, there are solvers. These packages are either 
+pure-Julia implementations of optimization algorithms, or they provide a Julia
+interface to external solvers (often written in C or C++). Each solver also 
+implements the API defined by MathOptInterface so that it can be used from JuMP.
 
 A non-exhaustive list of solvers available through JuMP is available in the
 [JuMP documentation](https://jump.dev/JuMP.jl/stable/installation/#Getting-Solvers-1).
+
+## JuMP extensions
+
+JuMP extensions are Julia packages which extend JuMP by providing additional
```

Comment:
"extends JuMP's domain-specific modeling language" ?
This would distinguish JuMP extensions from packages that uses JuMP.